doc/classes/SpinBox.xml

```diff
@@ -5,6 +5,19 @@
 	</brief_description>
 	<description>
 		SpinBox is a numerical input text field. It allows entering integers and floats.
+		[b]Example:[/b]
+		[codeblock]
+		func _ready():
+		  var spin_box = SpinBox.new()
+		  get_tree().get_root().add_child(spin_box)
+		  var line_edit = $SpinBox.get_line_edit()
+		  var is_enabled = false
+		  line_edit.set_context_menu_enabled(is_enabled)
```

Comment:
No need to save a boolean in a variable before using it, and here it would be better to access the property directly instead of its setter, so:
```
line_edit.context_menu_enabled = false
```

Reply:
I didn't even know you can do that!